vendor/github.com/openshift/api/config/v1/types_cluster_version.go

```diff
@@ -73,11 +73,16 @@ type ClusterVersionSpec struct {
 // progress, or is failing.
 // +k8s:deepcopy-gen=true
 type ClusterVersionStatus struct {
-	// current is the version that the cluster will be reconciled to. This
-	// value may be empty during cluster startup, and then will be set whenever
-	// a new update is being applied. Use the conditions array to know whether
-	// the update is complete.
-	Current Update `json:"current"`
+	// desired is the version that the cluster is reconciling towards.
```

Comment:
i think we might need an `ObservedGeneration` so that we can track `DesiredUpdate` change to `Desired`

Reply:
Ahh, didn't see https://github.com/openshift/cluster-version-operator/pull/63/files#diff-cf5e673e310efadb9cd9a2cbe7f58fdbR91